backstory/src/test/java/com/google/sps/images/VisionManagerTest.java

```diff
@@ -0,0 +1,73 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.images;
+
+import static org.mockito.Mockito.*;
+import com.google.cloud.vision.v1.AnnotateImageRequest;
+import com.google.cloud.vision.v1.AnnotateImageResponse;
+import com.google.cloud.vision.v1.BatchAnnotateImagesResponse;
+import com.google.cloud.vision.v1.EntityAnnotation;
+import org.mockito.ArgumentCaptor;
+import com.google.sps.images.AnnotatedImageTest;
+import com.google.cloud.vision.v1.ImageAnnotatorClient;
+import com.google.sps.images.VisionManager;
+import com.google.sps.images.data.AnnotatedImage;
+import org.junit.Assert;
+import org.junit.Test;
+import java.util.ArrayList;
+import java.util.List;
+import java.io.IOException;
+
+/**
+ * Tests for VisionManager.
+ */
+public final class VisionManagerTest {
+
+  /**
+   * Checkts that the request given to Vision API is valid
+   */
+  @Test
+  public void testCorrectRequestInput() throws IOException {
```

Comment:
Will you also be testing the output?

For example, in this test you could assign the return value to a List<AnnotatedImages> variable, and check that its contents are expected.

Reply:
I don't think we have to, because every remaining line of code in the method are method calls from Vision API, which are already tested.

Reply:
But it seems like the manager's handling of the response is untested then?